roles/forklift_versions/lookup_plugins/forklift_versions.py

```diff
@@ -0,0 +1,69 @@
+# python 3 headers, required if submitting to Ansible
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+      lookup: forklift_versions
+        author: Evgeni Golov <evgeni@redhat.com>
+        version_added: "0.9"
+        short_description: fetch Foreman/Katello versions from Forklift's versions.yaml
+        description:
+            - This lookup returns the component versions of a Foreman/Katello scenario
+        options:
+          file:
+            description: path to versions.yaml
+            required: True
+          scenario:
+            description: scenario to look up
+            required: True
+          scenario_version:
+            description: scenario version to look up
+        notes:
+          - this lookup will match the foreman version for the foreman scenatio
+          - this lookup will match the katello version for all other scenarios
+"""
+import yaml
+from ansible.errors import AnsibleError, AnsibleParserError
+from ansible.plugins.lookup import LookupBase
+
+try:
+    from __main__ import display
+except ImportError:
+    from ansible.utils.display import Display
+    display = Display()
+
+
+class LookupModule(LookupBase):
+
+    def run(self, terms, variables=None, **kwargs):
+        ret = []
+
+        for term in terms:
+            display.debug("Forklift lookup term: %s" % term)
+            lookup_params = {key: value for (key, value) in [x.split('=') for x in term.split()]}
+            scenario = lookup_params['scenario']
```

Comment:
Do we want to raise a nice error on KeyError?

Reply:
yes, I need to adapt ansible style error handling to this, the above is more of a draft :)